### PR TITLE
Fix UnitTest to be passed 

### PR DESCRIPTION
**Pull request by cruisediary**
- Change naming `import RxTests` to `RxTest`
- Add cocoapod RxOptional
- Fix unittest case testTitle to be passed

forked lastest master branch and used 2 space indent convention

### CODE REVIEW

File: RxTodoTests/Sources/Tests/TaskEditViewModelTests.swift [hunk 1]
```diff
@@ -33,12 +34,12 @@ class TaskEditViewModelTests: XCTestCase {
   func testTitle() {
     RxExpect { test in
       let viewModel = TaskEditViewModel(mode: .new)
-      test.assertNextEqual(viewModel.title.asDriver(), [""])
+        test.assertNextEqual(viewModel.title.asDriver().filterNil().map {$0}, [""])
```
Comment (devxoul) [inline RxTodoTests/Sources/Tests/TaskEditViewModelTests.swift]: I think this is an unnecessary indentation.

**Review by devxoul — CHANGES_REQUESTED** [review]

File: RxTodoTests/Sources/Tests/TaskEditViewModelTests.swift (hunk 1 above)
Comment (cruisediary) [reply RxTodoTests/Sources/Tests/TaskEditViewModelTests.swift]: hmm I think it is not work when compare `Variable<String?> to asDriver()` using assertNextEqual 

screen shot 2016-12-13 at 1 31 38 am

but if using `Variable<String> to asDriver()` work fine 
I think after  #7 TaskEditViewModel's title type changed to `Variable<String?>` this test broken 

how about using `test.assertNextEqual(viewModel.title.asDriver().filterNil(), [""])` then `test.assertNextEqual(viewModel.title.asDriver().filterNil().map {$0}, [""])` ? 
or is there a good solution to solve this issue :)

Comment (devxoul) [reply RxTodoTests/Sources/Tests/TaskEditViewModelTests.swift]: I mean, you have added 2 unnecessary spaces :grin:

Comment (cruisediary) [reply RxTodoTests/Sources/Tests/TaskEditViewModelTests.swift]: I misunderstood the comment 😭  

what do you prefer `.filterNil().map` or `.filterNil()` both way passed unittest

Comment (devxoul) [reply RxTodoTests/Sources/Tests/TaskEditViewModelTests.swift]: Absolutely `.filterNil()` because it's simple!

Comment (cruisediary) [reply RxTodoTests/Sources/Tests/TaskEditViewModelTests.swift]: :

> Absolutely .filterNil() because it's simple!

👍

**Review by devxoul — APPROVED** [review]

**Comment by devxoul** [comment]
Looks great :+1: Thank you for your contribution :tada:

**Pull request merged**